rkt/cli_apps.go

```diff
@@ -600,12 +600,49 @@ func (au *appName) Type() string {
 	return "appName"
 }
 
-// appAnnotation is for --user-annotation flags in the form of: --user-annotation=NAME=VALUE.
+// appAnnotation is for --annotation flags in the form of: --annotation=NAME=VALUE
```

Comment:
Can we do `--annotation=name=value` since for annotations the key has to be an ACIdentifier and can't be upper case.